lib/wasix/src/state/linker/module_handle.rs

```diff
@@ -0,0 +1,133 @@
+use derive_more::Debug;
+
+// IDs for special module handles
+// Not public to ensure they are only used through the safe API.
+// TODO: These should probably move to the C bindings at some point
```

Comment:
Note: these are internal to the linker and not visible to the C bindings at all.